dataclients/kubernetes/eastwest.go

```diff
@@ -43,3 +46,52 @@ func createEastWestRouteRG(name, ns, postfix string, r *eskip.Route) *eskip.Rout
 	ewr.Predicates = p
 	return ewr
 }
+
+func applyEastWestRange(eWRange map[string]string, host string, routes []*eskip.Route) {
+	for d, n := range eWRange {
```

Comment:
d is domain, n is cidr?
d seems fine, n is weird.

Reply:
should be network

Reply:
will change.